packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -200,6 +200,24 @@ const key = api.addApiKey('ApiKey', {
 });
 ```
 
+You can also import an API Key by it's Id to use with a usage plan.
+
+```ts
+const api = new apigateway.RestApi(this, 'hello-api', { });
+api.root.addMethod('GET'); // api must have atleast one method.
+
+const importedKey = apigateway.ApiKey.fromApiKeyId(this, 'imported-key', '<api-key-id>');
+
+const plan = api.addUsagePlan('UsagePlan', {
+  name: 'Easy',
+  apiKey: importedKey,
+  throttle: {
+    rateLimit: 10,
+    burstLimit: 2
+  }
+});
+```
```

Comment:
Keep only the relevant parts of a code snippet here.

````suggestion
```ts
const importedKey = ApiKey.fromApiKeyId(this, 'imported-key', '<api-key-id>');
```
````